ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -30,23 +30,29 @@
             let self = this;
             this._model.subscribe((action) => {
                 let state = action.target.getState();
-                let body = action.payload?.body;
-                if (body) {
-                    if (body.redirectUrl) {
-                        window.location.href = encodeURI(body.redirectUrl);
-                    } else if (body.thankYouMessage) {
-                        let formContainerElement = document.querySelector("[data-cmp-path='"+ self._path +"']");
-                        let thankYouMessage = document.createElement("div");
-                        thankYouMessage.setAttribute("class", "tyMessage");
-                        thankYouMessage.innerHTML = body.thankYouMessage;
-                        formContainerElement.replaceWith(thankYouMessage);
+                // execute the handler only if there are no rules configured on submitSuccess event.
+                if (!state.events.submitSuccess || state.events.submitSuccess.length === 0) {
+                    let body = action.payload?.body;
```

Comment:
This code should use the custom function which we have created, can you change it.

Reply:
We pas globals argument to defaultHandlers thorough rule-engine, if we directly call them in hardcoded handler then globals will not be available so we can't use them here. 
https://git.corp.adobe.com/livecycle/af-custom-functions/blob/main/src/index.js#L87

Reply:
Done